python/cuml/utils/input_utils.py

```diff
@@ -17,7 +17,7 @@
 import cuml.utils.numba_utils
 
 import cudf
-import cupy as cp
+# import cupy as cp
```

Comment:
Why comment instead of deleting? If you're trying to say "we'll add this back when packaging is resolved", I'd add that comment explicitly, otherwise just delete the line.